app/js/components/orderEntry/PastOrders.jsx

```diff
@@ -1,65 +1,33 @@
 import React from 'react';
+import axios from 'axios';
```

Comment:
axios is not been used anywhere in the code, you may consider removing this import statement

Reply:
Fixed